lib/report.js

```diff
@@ -107,11 +107,14 @@ class ConciseReporter extends Reporter {
         let value = res[key];
         if (key === 'message' && !value) return;
         if (key === 'actual' || key === 'expected') {
-          let type = typeof value;
-          if (type === 'object' && value !== null && value.constructor) {
-            type = value.constructor.name;
+          value = `${JSON.stringify(value)}`;
+          if (value !== 'undefined' && value !== 'null') {
+            let type = typeof value;
```

Comment:
`type` will always be equal a `'string'` here, meaning that everything inside this `if` block can be replaced with
```js
            value += `is string`;
```
without affecting the result.
Which probably means that the logic here is wrong.